app/models/manageiq/providers/redhat/infra_manager/ovirt_services/strategies/v3.rb

```diff
@@ -171,7 +171,7 @@ def start_clone(source, clone_options, phase_context)
 
     def vm_start(vm, cloud_init)
       vm.with_provider_object do |rhevm_vm|
-        rhevm_vm.start { |action| action.use_cloud_init(true) if cloud_init }
+        rhevm_vm.start { |action| action.use_cloud_init(true) if cloud_init.present? }
```

Comment:
Does this have the same caller as the method in the v4 strategy?  If so, the parameters need updating

Reply:
V3 do not support sysprep. The code is different.

Reply:
I guess my concern is that the v3 call is `vm_start(vm, Boolean)` and the v4 call is `vm_start(vm, Hash)`, so the caller needs to be aware of this if it's shared code.

Reply:
Please check here https://github.com/ManageIQ/manageiq-providers-ovirt/pull/91/files#diff-7f31c34b148696c2c255077a322e8e66R10. I changed it to have vm_start(vm, Hash) for both.